from .base import NotFoundException


class UserNotFoundException(NotFoundException):
    pass
